examples/aks/stack.tf

```
@@ -0,0 +1,192 @@
+
+  kubernetes {
+    host                   = module.cluster.admin_host
+    cluster_ca_certificate = base64decode(module.cluster.admin_cluster_ca_certificate)
+    client_key             = base64decode(module.cluster.admin_client_key)
+    client_certificate     = base64decode(module.cluster.admin_client_certificate)
+  }
+}
+
+module "ingress" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-traefik.git//aks?ref=v1.2.1"
+
+  cluster_name     = local.cluster_name
+  base_domain      = azurerm_dns_zone.this.name
+  argocd_namespace = module.argocd.argocd_namespace
+
+  enable_service_monitor       = false
+  node_resource_group_name     = module.cluster.node_resource_group
+  dns_zone_resource_group_name = azurerm_resource_group.default.name
+}
+
+module "azure-workload-identity" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-azure-workload-identity.git?ref=v0.1.0"
+
+  argocd_namespace = module.argocd.argocd_namespace
+
+  azure_tenant_id = data.azuread_client_config.current.tenant_id
+}
+
+module "cert-manager" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-cert-manager.git//aks?ref=v4.0.0"
+
+  cluster_name     = local.cluster_name
+  base_domain      = azurerm_dns_zone.this.name
+  argocd_namespace = module.argocd.argocd_namespace
+
+  cluster_oidc_issuer_url      = module.cluster.oidc_issuer_url
+  node_resource_group_name     = module.cluster.node_resource_group
+  dns_zone_resource_group_name = azurerm_resource_group.default.name
+  enable_service_monitor       = false
+
+  dependency_ids = {
+    azure-workload-identity = module.azure-workload-identity.id
+  }
+}
+
+module "loki-stack" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-loki-stack.git//aks?ref=v2.2.0"
+
+  argocd_namespace = module.argocd.argocd_namespace
+
+  distributed_mode = true
+
+  logs_storage = {
+    container                        = azurerm_storage_container.logs.name
+    storage_account                  = azurerm_storage_account.this.name
+    managed_identity_node_rg_name    = module.cluster.node_resource_group
+    managed_identity_oidc_issuer_url = module.cluster.oidc_issuer_url
+  }
+
+  dependency_ids = {
+    azure-workload-identity = module.azure-workload-identity.id
+  }
+}
+
+module "thanos" {
+  source = "git::https://github.com/camptocamp/devops-stack-module-thanos.git//aks?ref=v1.0.0"
+
+  cluster_name     = local.cluster_name
+  base_domain      = azurerm_dns_zone.this.name
+  argocd_namespace = module.argocd.argocd_namespace
+  cluster_issuer   = "letsencrypt-staging"
+
+  metrics_storage = {
+    container           = azurerm_storage_container.metrics.name
+    storage_account     = azurerm_storage_account.this.name
+    storage_account_key = azurerm_storage_account.this.primary_access_key
+  }
+
+  thanos = {
+    oidc = local.oidc
+  }
+
+  dependency_ids = {
+    cert-manager = module.cert-manager.id
+  }
+}
+
+module "monitoring" {
```

```suggestion
module "kube-prometheus-stack" {
```
Just a question of style, but on other examples I started naming the modules with the name of the module itself. The argument I have for standardizing the names on the module declarations, is that it would be easier to understand the documentation on the dependencies throughout each module documentation page. [See Argo CD for example](https://github.com/camptocamp/devops-stack-module-argocd#dependencies).